### DISCUSSION TITLE
self-hosting/installation/

### DISCUSSION THREAD

**Discussion by giscus** [discord-tickets/bot#462] [Documentation]
# self-hosting/installation/

The most popular open-source ticket management bot for Discord.

https://discordtickets.app/self-hosting/installation/

**Comment by ElectroGamerzz** [comment]
Hi I need help

**Comment by ElectroGamerzz** [comment]
Can you Explain me About this
Error: The "DB_CONNECTION_URL" environment variable must be set when "DB_PROVIDER" is not "sqlite".

**Reply by eartharoid** [reply]
You need to either set the database provider to sqlite, or set it to mysql/postgresql and also set the URL.
Set these variables in the .env file if you're not setting them directly.

**Comment by ElectroGamerzz** [comment]
I hope that someone explain me about this my Discord name is this **its_wali**

**Comment by RooRay** [comment]
If you want faster support, I'd recommend joining the official support server for Discord Tickets [here](https://lnk.earth/discord)
